### PR TITLE
Adding bookmarks and various TLC

### PR DESCRIPTION
**Pull request by aidankmcl**

### CODE REVIEW

File: cluster/app.py [hunk 1]
```diff
@@ -202,7 +203,7 @@ def post(self):
                 cluster_info['places'] = create_full_cluster_list(cluster_info, 'places')
                 cluster_info['organizations'] = create_full_cluster_list(cluster_info, 'organizations')
 
-                all_keywords = create_full_cluster_dict(cluster_info, 'keywords')
+                all_keywords = create_full_cluster_dict(cluster_info, 'title_keywords')
```
Comment (sihrc) [inline cluster/app.py:154]: What about normal keywords? Is it done somewhere else

File: cluster/app.py [hunk 2]
```diff
@@ -233,10 +234,41 @@ def post(self):
...
 
 
+class BookmarkHandler(tornado.web.RequestHandler):
+
+    def get(self):
+        """shows added images"""
+        session = DBSession()
+        bookmarks = session.query(Bookmark.search).distinct(Bookmark.search)
+        query_groups = [bookmark[0] for bookmark in bookmarks]
+        print query_groups
```
Comment (sihrc) [inline cluster/app.py]: some prints still

**Comment by sihrc** [comment]
The clean-up code branch removed SQLAlchemy stuff because we weren't using it. We should rebase this branch onto that one and keep only the bits of SQLalchemy we need.

**Comment by sihrc** [comment]
Aside from ^, just a couple of print statements left over & 1 comment

File: cluster/app.py (hunk 1 above)
Comment (aidankmcl) [inline cluster/app.py:154]: Right now we're using title_keywords for cluster titles

**Comment by sihrc** [comment]
LGTM

**Pull request merged**